src/lcs.nim

```diff
@@ -29,18 +28,23 @@ proc runMain() =
   if licenseType == "":
     # exit cli if type is not set
     echo "License type flag required! (--type, -t)"
-    quit(0)
+    quit(1)
 
-  try:
-    license = licenses[licenseType]
-  except IndexDefect:
+  # Get license contents
+  var license = licenses.getOrDefault(licenseType, "")
```

Comment:
We can use `getOrDefault` instead of checking if an exception will be raised to get license contents or an empty string if there's no such license